projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -333,6 +333,11 @@ public static function parse_dimensions_from_filename( $src ) {
 	 * @return string
```

Comment:
I typically use `string|mixed` for filtered params and here for the return type:

```suggestion
	 * @return string|mixed
```

Reply:
Heh, I just pushed up a commit to update that since static analysis also complained about that.

Reply:
Actually, what I typically do is fix the type of the param so that the function can have a fixed PHP type hint:

```php
/**
 * Filters foo.
 *
 * @param string|mixed $foo Foo.
 * @return string Foo.
 */
function filter_foo( $foo, int $post_id ): string {
	if ( ! is_string( $foo ) ) {
		// Because plugins do bad things.
		$foo = '';
	}

	 // Filtering logic goes here.

	 return $foo;
}
add_filter( 'foo', 'filter_foo', 10, 2 );
```